app/controllers/projects_controller.rb

```diff
@@ -28,17 +28,27 @@ def new
 
   # GET /projects/1/edit
   def edit
+    respond_to do |format|
+      format.html { render :edit }
+      format.json do
+        render json: { project: @project }
+      end
+    end
   end
 
   # POST /projects
-  def create
+  def create # rubocop:disable Metrics/MethodLength
```

Comment:
Incorrect response when trying to create an empty project:
![image](https://user-images.githubusercontent.com/35406008/204102824-e3d9f506-b269-4334-b6e4-1a37547e4f02.png)